pkg/cmd/opts/gocloud_integration_test.go

```diff
@@ -0,0 +1,36 @@
+// +build integration
+
+package opts_test
+
+import (
+	"path/filepath"
+	"testing"
+
+	"github.com/jenkins-x/jx/pkg/cmd/testhelpers"
+	"github.com/stretchr/testify/assert"
+
+	"github.com/jenkins-x/jx/pkg/cmd/clients"
+	"github.com/jenkins-x/jx/pkg/cmd/opts"
+	"github.com/jenkins-x/jx/pkg/log"
+)
+
+func TestCommonOptions_EnsureGCloudBinaryInstalled(t *testing.T) {
+	o := opts.NewCommonOptionsWithFactory(clients.NewFactory())
+	o.NoBrew = true
```

Comment:
Since we're not having brew checked on installation, I'm not sure why we need this.

Reply:
Yeah removing :+1: